### ISSUE TITLE
download the xml file based on the publish date

### ISSUE THREAD

**Issue by Spatial-R** [dami82/easyPubMed#5]
Hello, thanks for your nice package. Recently, I want to download all the articles in the Pubmed between 2019 and 2020.  Thus, I download the article by date using the batch_pubmed_download function. The example code is as followed:

`quary_str <- paste0("2019/01/01","[ppdt] AND journal article[pt]")
  batch_pubmed_download(pubmed_query_string = quary_str,dest_dir="Temp",
                        format = "xml");
`

I have checked the results and found that the downloaded article is inconsistent with the right publish date.  Is there any way to solve this problem? Thanks

**Comment by dami82** [comment]
Do you have the same issue when you run the same query using the web interface?

**Comment by Spatial-R** [comment]
Yes. Actually, I want to download all the PubMed data to extract the published date and received date. Therefore, I used the publish date as the query id to download all the data. Is there any other way to finish this project?
